commitlint.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   rules: {
     'subject-case': [1, 'always', 'sentence-case'],
     'body-max-line-length': [0],
+    'footer-max-line-length': [0],
```

Comment:
This was being flagged on my commit message (probably confused by the colons). Disabled it as we haven't been using footers.